### PR TITLE
Fix issue 6945

### LINKED ISSUE #6945: bug report:NPE
- Why do you submit this issue?
- [ ] Question or discussion
- [x] Bug
- [ ] Requirement
- [ ] Feature or performance improvement
___

In method `org.apache.skywalking.oap.server.configuration.configmap.ConfigmapConfigurationWatcherRegister#readConfig`:

NullPointException occurs when method `io.kubernetes.client.openapi.models.V1ConfigMap#getData` returns null,The method `io.kubernetes.client.openapi.models.V1ConfigMap#getData` returns empty, which is normal for k8s;

### PR DESCRIPTION
**Pull request by JoeKerouac**
Fix #6945

### CODE REVIEW

**Comment by wu-sheng** [comment]
You need to update CHANGES.md in the root path.

**Comment by JoeKerouac** [comment]
> You need to update CHANGES.md in the root path.

ok

File: CHANGES.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Release Notes.
 * Fix: Envoy error logs are not persisted when no metrics are generated
 * Fix: Memory leakage of low version etcd client. [fix-issue](https://github.com/jurmous/etcd4j/pull/185)
 * Allow multiple definitions as fallback in metadata-service-mapping.yaml file.
+* Fix: NPE when configmap has no data. [fix-issue](https://github.com/apache/skywalking/pull/6948)
```
Comment (wu-sheng) [inline CHANGES.md:44]:
```suggestion
* Fix: NPE when configmap has no data. 
```

SkyWalking native issue should not be linked in the changelog. Our git commit log already provides this.

Comment (JoeKerouac) [reply CHANGES.md]: ok, Thx.

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**